### PR TITLE
Feature/client

### PR DESCRIPTION
**Pull request by andreyPromaster**

### CODE REVIEW

File: tests/unit/test_story.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class TestStoryDataStore:
+    def test_get_story(self, mock_db_session, test_data):
+        story = mock_db_session.get_story("test_id")
+        print(story.id, test_data)
+        assert story.id == test_data["id"]
+        assert story.root == test_data["root"]
+
+    def test_get_not_exist_story(self, mock_db_session):
+        story = mock_db_session.get_story("not_exist_id")
+        assert story is None
```
Comment (vadim-denisov) [inline tests/unit/test_story.py:19]: This should work without class
```suggestion
def test_get_not_exist_story(mock_db_session):
    story = mock_db_session.get_story("not_exist_id")
    assert story is None
```

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+
+@pytest.fixture(scope="session")
+def test_data():
+    with open("tests/test_data.json", "r") as file:
+        test_data = json.load(file)
+        return test_data
+
+
+@pytest.fixture
```
Comment (vadim-denisov) [inline tests/conftest.py:47]: What difference between `@pytest.fixture` and `@pytest.fixture()` above?

File: tests/unit/test_story.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import pytest
+
+from common.entities.schemas import Node
+from utilities.exceptions import DynamoDBError
+
+
+def test_get_story(mock_db_session, test_data):
+    story = mock_db_session.get_story("test_id")
+    print(story.id, test_data)
```
Comment (Vladgasik) [inline tests/unit/test_story.py:11]:
```suggestion
```

File: client/__main__.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import asyncio
```
Comment (vadim-denisov) [inline client/__main__.py:1]: What is the advantages of using `asyncio` here?

Comment (andreyPromaster) [reply client/__main__.py:1]: Are you recommend to create function and call it in this file?

Comment (vadim-denisov) [reply client/__main__.py:1]: I'm just asking why should we use asyncio for client tool?

Comment (andreyPromaster) [reply client/__main__.py:1]: `asyncio` needs to run async function. Mb in def `__main__`. It is redundant. I will remove `asyncio` calls in def `start_game`.

Comment (vadim-denisov) [reply client/__main__.py:1]: No, I'm asking in general. Why should we use async approach instead of sync approach?

Comment (andreyPromaster) [reply client/__main__.py:1]: Oh, in this side, I chose this way only to call to mind  async approach

**Pull request merged**